src/utils/frame.js

```diff
@@ -97,3 +98,11 @@ export function formatDisplayName(frame: Frame) {
   displayName = simplifyDisplayName(displayName);
   return endTruncateStr(displayName, 25);
 }
+
+export function formatCopyName(frame: Frame) {
+  const displayName = formatDisplayName(frame);
+  const fileName = getFilename(frame.source);
+  const frameLocation = frame.location.line;
+
+  return `${displayName}  ${fileName}#${frameLocation}`;
+}
```

Comment:
lets keep this on the top of the component `Frame` because it is kinda like rendering